### PR TITLE
Bumps

### PR DESCRIPTION
**Pull request by yeayeah**
bump tor, ngircd and openvpn to new versions

### CODE REVIEW

File: pkg/openvpn [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [mirrors]
-https://swupdate.openvpn.org/community/releases/openvpn-2.5.0.tar.xz
+https://swupdate.openvpn.org/community/releases/openvpn-2.5.2.tar.gz
```
Comment (rofl0r) [inline pkg/openvpn:2]: they don't offer .xz download anymore ?

File: pkg/ngircd [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-sha512=4a8fbf14a0e0e3a128f97ee5c440dd52a7d02e1b518833870dce7894649faad397112f4a60629d3b10d9c62fc7a783498109406116a3adb35cfd6cadf3b16cbc
-pkgver=1
+filesize=371716
+sha512=5fe052096994d0cb9488962a9c8e0602442536454dba9aa3922903aa3c07e612fd125bc6dfb0717e575aeb327c87f6d690982fa4744def7b6b95320324a93a17
+pkgver=2
 desc="ngIRCd - Next Generation IRC Server"
 service=ngircd
 serviceflags="--down --log"
-tardir=ngircd-25~rc1
+tardir=ngircd-26
```
Comment (rofl0r) [inline pkg/ngircd:11]: tardir line can probably be removed here since it's identical to tarball prefix

**Comment by yeayeah** [comment]
thanks, changes have been applied

**Comment by rofl0r** [comment]
thanks, looks good now. can you remove the draft status?

**Pull request closed without merging**